debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/TableSnapshotWriterCSV.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.server.ybexporter;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TableSnapshotWriterCSV implements RecordWriter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TableSnapshotWriterCSV.class);
+    private ExportStatus es;
+    private String dataDir;
+    private Table t;
+    private CSVPrinter csvPrinter;
+
+    public TableSnapshotWriterCSV(String datadirStr, Table tbl) {
+        dataDir = datadirStr;
+        t = tbl;
+
+        var fileName = getFullFileNameForTable();
+        try {
+            var f = new FileWriter(fileName);
+            csvPrinter = new CSVPrinter(f, CSVFormat.POSTGRESQL_CSV);
```

Comment:
Can we use `CSVFormat.POSTGRESQL_TEXT`?

Reply:
Will explore/test TEXT format in my work on optimizations. 
Is the motivation strictly performance based? Or any other reason why we'd prefer TEXT over CSV?

Reply:
Tested out TEXT format. For some reason, the format isn't working as expected - using COPY command with format TEXT wasn't working. Had to eventually use CSV with specific options to get it to work. (`COPY test_basic FROM '/tmp/export-dir/test_basic_data.sql'  CSV DELIMITER E'\t' ESCAPE '\' NULL '\N';`)